### PR TITLE
PROJ-54 : forklar figur

### PR DESCRIPTION
**Pull request by jenspetur94**

### CODE REVIEW

File: sections/sprint-1/prototypes.tex [hunk 1]
```diff
@@ -30,7 +30,10 @@ \section{Prototypes}
 \end{figure}
 
 When all users have connected, the host can start the game, and they will now see the virtual game field, as seen on \autoref{fig:prototype:ingame}.
-In this prototype, the player's icon is highlighted by having a solid color, whereas the other players are just shown as outlines.
+In this prototype the players icons are represented by small squares and the goals are represented by larger rectangles.
```
Comment (Tekrus) [inline sections/sprint-1/prototypes.tex]:
```suggestion
In this prototype, the players' icons are represented by small squares and the goals are represented by larger rectangles.
```

File: sections/sprint-1/prototypes.tex [hunk 2]
```diff
@@ -30,7 +30,10 @@ \section{Prototypes}
...
+The current player's icon is highlighted by having a solid color, whereas the other players are just shown as outlines. 
+The players are divided into two teams which are represented by the color of the player icons.
+The goals have the same color as the players that they belong to.
```
Comment (Tekrus) [inline sections/sprint-1/prototypes.tex]:
```suggestion
The goals are indicated by the long boxes and have the same color as the players that they belong to.
```

**Review by Tekrus — CHANGES_REQUESTED** [review]

File: sections/sprint-1/prototypes.tex [hunk 3]
```diff
@@ -30,7 +30,10 @@ \section{Prototypes}
 \end{figure}
 
 When all users have connected, the host can start the game, and they will now see the virtual game field, as seen on \autoref{fig:prototype:ingame}.
-In this prototype, the player's icon is highlighted by having a solid color, whereas the other players are just shown as outlines.
+In this prototype, the players' icons are represented by small squares and the goals are represented by larger rectangles.
```
Comment (Tekrus) [inline sections/sprint-1/prototypes.tex]:
```suggestion
In this prototype, the players' icons are represented by small squares and the goals are represented by larger rectangles and have the same color as the players they belong to.
```

File: sections/sprint-1/prototypes.tex [hunk 4]
```diff
@@ -30,7 +30,10 @@ \section{Prototypes}
...
+The current player's icon is highlighted by having a solid color, whereas the other players are just shown as outlines. 
+The players are divided into two teams which are represented by the color of the player icons.
+The goals are indicated by the long boxes and have the same color as the players that they belong to.
```
Comment (Tekrus) [inline sections/sprint-1/prototypes.tex]:
```suggestion
```

**Review by Tekrus — CHANGES_REQUESTED** [review]

File: sections/sprint-1/prototypes.tex (hunk 3 above)
Comment (Eduardsen) [reply sections/sprint-1/prototypes.tex]: that have the same color as the players they belong to.

Comment (Eduardsen) [reply sections/sprint-1/prototypes.tex]: 😡

**Review by Tekrus — APPROVED** [review]

**Review by Eduardsen — APPROVED** [review]

**Pull request merged**